script/cibuild

```diff
@@ -22,11 +22,16 @@ function loadEnvironmentVariables(filePath) {
 }
 
 function readEnvironmentVariables() {
-  if (process.platform === 'win32')
+  if (process.platform === 'win32') {
     loadEnvironmentVariables(path.resolve('/jenkins/config/atomcredentials'));
-  else if (process.platform === 'darwin') {
+  } else if (process.platform === 'darwin') {
     loadEnvironmentVariables('/var/lib/jenkins/config/atomcredentials');
     loadEnvironmentVariables('/var/lib/jenkins/config/xcodekeychain');
+  } else if (process.platform === 'linux') {
+    // Use clang for building native code, the GCC on precise is too old.
+    process.env['CC'] = 'clang';
```

Comment:
Super minor, but this could be rewritten as `process.env.CC` to be consistent with other env property access in this file.